app/src/main/java/com/evrencoskun/tableviewsample/tableview/popup/ColumnHeaderLongPressPopup.java

```diff
@@ -115,6 +115,8 @@ public boolean onMenuItemClick(MenuItem menuItem) {
             case SCROLL_ROW:
                 mTableView.scrollToRowPosition(5);
                 break;
+            default:
+                break;
```

Comment:
Shouldn't you return `false` here? Same in the other file.

Reply:
Indeed, `false` should be returned here. I'll fix it.